ADAL/tests/integration/AADAuthorityValidationIntegrationTests.m

```diff
@@ -54,35 +54,57 @@ - (void)tearDown
 }
 
 //Does not call the server, just passes invalid authority
-- (void)testValidateAuthorityError
+
+
+
+- (void)testValidateAuthority_whenSchemeIsHttp_shouldFailWithError
```

Comment:
As far as I understand, the test name should be something like this testValidateAuthority_whenSchemeIsHttp_**shouldReturnValidatedFalseErrorNil**

According to our test doc:
> **Result** should describe the result of the method execution that test is checking with asserts